### PR TITLE
AdjustedNServiceBus.AcceptanceTesting nuspec

### LINKED ISSUE #2817: Change NServiceBus.AcceptanceTesting package title
The AcceptanceTesting NuGet package has the following title and description:

```
<title>NServiceBus Acceptance Tests</title>
<description>Acceptance tests for nservicebus core functionality</description>
```

when searching for the AcceptanceTesting library on NuGet this is slightly confusing. To better distinct it from the acceptance tests source package and clarify the package's purpose we should change these fields.

### PR DESCRIPTION
**Pull request by timbussmann**
fixes #2817 

* title should represent the package id more appropriately
* add a note that this isn't an officially supported package

@Particular/nservicebus-maintainers please review

### CODE REVIEW

File: packaging/nuget/nservicebus.acceptancetesting.nuspec [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <projectUrl>$projectUrl$</projectUrl>
     <iconUrl>$iconUrl$</iconUrl>
     <requireLicenseAcceptance>$requireLicenseAcceptance$</requireLicenseAcceptance>
-    <description>Acceptance tests for nservicebus core functionality</description>
+    <description>Acceptance Testing Framework for NServiceBus endpoints. This is an unofficial and unsupported package.</description>
```
Comment (bording) [inline packaging/nuget/nservicebus.acceptancetesting.nuspec]: "Testing" and "Framework" probably shouldn't be capitalized here.

**Comment by bording** [comment]
Should we also make some similar tweaks to the NServiceBus.AcceptanceTests.Sources package?

File: packaging/nuget/nservicebus.acceptancetesting.nuspec (hunk 1 above)
Comment (danielmarbach) [inline packaging/nuget/nservicebus.acceptancetesting.nuspec]: Not sure we need unofficial and unsupported in the same sentence. How about

> This is an unsupported package. Breaking changes might be introduced without following semantic versioning.

or maybe just

> This is an unsupported package.

But that begs the question why we would ever release that on nuget?

Comment (bording) [inline packaging/nuget/nservicebus.acceptancetesting.nuspec]: > But that begs the question why we would ever release that on nuget?

I was curious about that as well.

**Comment by timbussmann** [comment]
@danielmarbach @bording adjusted description.

> Should we also make some similar tweaks to the NServiceBus.AcceptanceTests.Sources package?

The title seems to fit for the sources packages. We should probably update the description, but it really seems like we should remove the sources package from nuget?

**Comment by bording** [comment]
>The title seems to fit for the sources packages

The title should probably be the description instead. The title for the sources package makes things look really awkward on nuget.org:

https://www.nuget.org/packages/NServiceBus.AcceptanceTests.Sources/



> it really seems like we should remove the sources package from nuget?

We could just push it up to myget instead. I wonder if anyone else finds it useful?

**Comment by danielmarbach** [comment]
I agree we should remove the source package from nuget and also the AcceptanceTesting package. Thoughts @andreasohlund ?

**Comment by timbussmann** [comment]
I fully agree, but please open a new issue about the source package.

**Comment by danielmarbach** [comment]
It improves the situation, so let's go

**Comment by andreasohlund** [comment]
I'm fine with keeping them on MyGet since community transports can get them from there (eg the SQS transport)

**Comment by danielmarbach** [comment]
Do we need to change the deployment on octopus?

**Comment by timbussmann** [comment]
raised an issue here: https://github.com/Particular/NServiceBus/issues/3592

**Pull request merged**